packages/fxa-settings/src/pages/PostVerify/ThirdPartyAuthCallback/index.tsx

```diff
@@ -124,11 +124,19 @@ const ThirdPartyAuthCallback = ({
     }
 
     try {
+      const fxaParams = integration.getFxAParams();
+      let originalService;
+      if (fxaParams) {
+        const params = new URLSearchParams(fxaParams);
+        if (params) {
+          originalService = params.get('service') || params.get('client_id');
```

Comment:
Non-blocking question: would there be a difference between using `client_id` first if that exists and then `service` (albeit in that case we do want `client_id` vs this one)?

I see one other usage where it's the other-way around: https://github.com/mozilla/fxa/blob/23f8f3f1d78565d7bedc005a96a384d65b4d1b69/packages/fxa-settings/src/models/hooks.ts#L150

Reply:
I'm not sure? I've tested this with OAuth flows and Sync flows that might possibly change it and works as expected.